index.html

```diff
@@ -39,9 +39,9 @@
               </li>
               <li>
                 <ul class="iwq ">
-                  <li><a class="mobile-nav-links" herf="#">Portoflio</a></li>
-                  <li><a class="mobile-nav-links" href="#">About</a></li>
-                  <li><a class="mobile-nav-links" href="#">Contact</a></li>
+                  <li><a class="mobile-nav-links" href="#portfolio">Portoflio</a></li>
```

Comment:
- [ ] your website looks fantastic I would love to suggest adding a smoother animation to your website's scroll behavior, try to add `scroll-behavior: smooth;` to your HTML tag in CSS it works like magic! 😮 

``` css
html {
scroll-behavior: smooth;
}
```

Before           | After
:-------------------------:|:-------------------------:
![before](https://user-images.githubusercontent.com/89282221/189975888-cd9e04f1-b6a9-4e1e-a5de-bdd05b6d8931.gif) | ![mAFTER](https://user-images.githubusercontent.com/89282221/189975757-65de4564-3fc4-40b9-b40e-1bf3919546b8.gif)

Reply:
- Thanks for your review 😊I already made the page moves by scroll behavior,  maybe you didn't notice that you can check this image to make sure from what I am saying. 👎 


![Screenshot 2022-10-12 180123](https://user-images.githubusercontent.com/83360636/195392849-00f5a7f8-13ba-406b-849b-7038662e95f6.png)